src/test/java/org/elasticsearch/gateway/local/RecoveryBackwardsCompatibilityTests.java

```diff
@@ -77,7 +77,8 @@ public void testReusePeerRecovery() throws Exception {
         indexRandom(true, builders);
         ensureGreen();
         if (randomBoolean()) { // just make sure it doesn't break anything - we seal before we actually bump replicas
-            backwardsCluster().internalCluster().client().admin().indices().prepareSealIndices("test").get();
+            logger.info("--> trying to sync flush");
+            assertEquals(SyncedFlushUtil.attemptSyncedFlush(internalCluster(), "test").failedShards(), 0);
```

Comment:
don't we need backwardsCluster().internalCluster() here? also, are we sure we have at least one local (client) node? o.w. there is not SyncedFlushtService instance..